genanki/__init__.py

```diff
@@ -243,12 +276,67 @@ def _format_tags(self):
     return ' ' + ' '.join(self.tags) + ' '
 
 
+class OptionsGroup:
+  def __init__(self, options_id=None, name=None):
```

Comment:
I'd like to follow the pattern of allowing all arguments to be set via keyword arguments to `__init__`.

So you should write something like
```python
def __init__(
    self,
    options_id=None,
    options_group_name=None,
    max_time_per_answer=60,
    ...
    misc=0):
  self.options_id = options_id
  self.options_group_name = options_group_name
  ...
```

I realize this is extremely verbose, so you could also go this alternate route:
```python
class OptionsGroup:
  ATTRIBUTES_WITH_DEFAULTS = {
    'options_id': None,
    'options_group_name': None,
    'max_time_per_answer': 60,
    ...
  }

  def __init__(self, **kwargs):
    for attr, default in self.ATTRIBUTES_WITH_DEFAULTS.items():
      setattr(self, attr, kwargs.get(attr, default))
```
Either style is fine. However, the first style is advantageous in that it's easier for tools like IDEs to introspect. And it's pretty easy to automatically generate the code so it ends up being about the same amount of work.

Reply:
Will do!